clients/python/test_python_client.py

```diff
@@ -111,7 +111,8 @@ def test_python_client():
             print(api_response)
         except airflow_client.client.exceptions.OpenApiException as e:
             print(f"[red]Exception when calling DAGRunAPI->post_dag_run: {e}\n")
-            errors = True
+            # TODO(pierrejeambrun): We need to fix post_dag_run to not return 422
```

Comment:
I left this TODO @pierrejeambrun  (or maybe others) - because post_dag_run produced 422 error.

But I also think there is something unfinished here - and that releates to what @jscheffl wrote about. 

We still have `api_connexion` and we still use the old manually written v1.yml to generate python client and I think it should be changed (but maybe in a follow-up PR?) @jedcunningham  @bbovenzi @pierrejeambrun ?

Reply:
Yes, let's do it in a follow up.